Rlabkey/DESCRIPTION

```diff
@@ -1,5 +1,5 @@
 Package: Rlabkey
-Version: 2.3.4
+Version: 2.3.5
 Date: 2020-01-31
```

Comment:
Should also bump the date.

Reply:
Also, the version number and date are listed in the Rlabkey-package.Rd file and should be updated there as well.

Reply:
good point, thank you.